components/developer/swig/Makefile

```diff
@@ -79,23 +79,17 @@ COMPONENT_TEST_TARGETS = \
         partialcheck-python-test-suite \
         partialcheck-ruby-test-suite
 
-# common targets
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
-
-test:		$(TEST_32_and_64)
-
+# Build dependencies
+PERL_REQUIRED_PACKAGES += runtime/perl
 REQUIRED_PACKAGES += developer/java/openjdk7
 REQUIRED_PACKAGES += developer/parser/bison
```

Comment:
Doesn't openjdk11 work?  runtime/java(openjdk11 should provide everything needed. We might be able to obsolete openjdk7.